graphql_compiler/tests/test_ir_generation_errors.py

```diff
@@ -1050,3 +1052,173 @@ def test_invalid_edge_degree_queries(self):
         for invalid_graphql in invalid_queries:
             with self.assertRaises(GraphQLCompilationError):
                 graphql_to_ir(self.schema, invalid_graphql)
+
+    def test_missing_directives_in_schema(self):
+        """The schema should define all directives which are supported by the graphql compiler,
```

Comment:
Please use the first line of the docstring as a one-line summary/headline, and move the rest of the explanation and detail into the body of the docstring.